test/util/sdkutil/proposals.go

```diff
@@ -0,0 +1,35 @@
+package sdkutil
+
+import (
+	"fmt"
+
+	blobtypes "github.com/celestiaorg/celestia-app/x/blob/types"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+// MaxBlockBytesParamChange returns a param change for the max block bytes
+// consensus params.
+func MaxBlockBytesParamChange(codec codec.Codec, maxBytes int) proposal.ParamChange {
```

Comment:
Could this be used in other places?

Reply:
yeah, we're using it in the sdk test and the square size test